.github/workflows/push.yml

```diff
@@ -73,17 +61,4 @@ jobs:
       if: github.repository == 'vmware-tanzu/velero-plugin-for-gcp'
       run: |
         docker login -u ${{ secrets.DOCKER_USER }} -p ${{ secrets.DOCKER_PASSWORD }}
-        VERSION=$(./hack/docker-push.sh | grep 'VERSION:' | awk -F: '{print $2}' | xargs)
-
-        # Upload velero-plugin-for-gcp image package to GCS
-        source hack/ci/build_util.sh
-        BIN=velero-plugin-for-gcp
-        GCS_BUCKET=velero-builds
-        VELERO_PLUGIN_IMAGE=${BIN}-${VERSION}
-        VELERO_PLUGIN_IMAGE_FILE=${VELERO_PLUGIN_IMAGE}.tar.gz
-        VELERO_PLUGIN_IMAGE_BACKUP_FILE=${VELERO_PLUGIN_IMAGE}-'build.'${GITHUB_RUN_NUMBER}.tar.gz
-
-        cp ${VELERO_PLUGIN_IMAGE_FILE} ${VELERO_PLUGIN_IMAGE_BACKUP_FILE}
-
-        uploader ${VELERO_PLUGIN_IMAGE_FILE} ${GCS_BUCKET}
-        uploader ${VELERO_PLUGIN_IMAGE_BACKUP_FILE} ${GCS_BUCKET}
+        ./hack/docker-push.sh
```

Comment:
How does docker-push.sh push to GCR?  I see the last line in the script is calling "make container"

Reply:
https://github.com/vmware-tanzu/velero-plugin-for-gcp/blob/04062a61b5baafd1859f74327edb1d155e90086f/hack/docker-push.sh#L99

that is how. When you build with this output type, the result is a pushed image.